cmake/VerifyCompiler.cmake

```diff
@@ -35,11 +35,11 @@ elseif(HIP_PLATFORM STREQUAL "hcc")
     else()
         # Workaround until hcc & hip cmake modules fixes symlink logic in their config files.
         # (Thanks to rocBLAS devs for finding workaround for this problem.)
-        list(APPEND CMAKE_PREFIX_PATH /opt/rocm/hcc /opt/rocm/hip)
-        # Ignore hcc warning: argument unused during compilation: '-isystem /opt/rocm/hip/include'
+        list(APPEND CMAKE_PREFIX_PATH ${CMAKE_INSTALL_PREFIX}/hcc ${CMAKE_INSTALL_PREFIX}/hip)
```

Comment:
CMAKE_INSTALL_PREFIX should not be used to search for tools required to build the library. This shold be a separate variable.

Reply:
Yes will add ROCM_PATH similar to rocPRIM changes.